### PR TITLE
Update peerDependencies to be compatible with React 0.13

### LINKED ISSUE #40: React 0.13
@ForbesLindesay we need to update `react` `peerDependency` to `~0.13.0`.

BTW, npm team was deprecated `peerDependency`.

### PR DESCRIPTION
**Pull request by ai**
Too fix #40

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,12 +19,12 @@
...
     "istanbul": "^0.3.2",
     "marked": "~0.3.2",
-    "react": "~0.12.0",
+    "react": "~0.13.0",
     "rimraf": "~2.2.8",
     "testit": "~1.2.0"
   },
   "peerDependencies": {
-    "react": "~0.12.0"
+    "react": ">=0.12.0 <0.14.0"
```
Comment (TimothyGu) [inline package.json]: Why not ~0.13.0?

Comment (ai) [inline package.json]: Just `~0.13` will no include 0.12. `>=0.12 ~0.13` is more confused, because it mix different types (is it even will works?)

Comment (ForbesLindesay) [inline package.json]: I think this is perfect.  It makes it crystal clear what ranges we are accepting, and very easy to update.

**Comment by ForbesLindesay** [comment]
I will try and publish this update to npm tonight (if nobody else gets round to it).  In the mean time you can get this module via:

```
npm install jadejs/react-jade
```

File: package.json (hunk 1 above)
Comment (TimothyGu) [inline package.json]: LGTM, I misread the line.

**Pull request merged**